clients/ts/FunctionalTests/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Kestrel": {
+        "Certificates": {
+            "Default": {
+                "Path": "testcert.pfx",
+                "Password": "testPassword"
+            }
+        },
+        "EndPointDefaults": {
+            "Protocols": "Http1"
```

Comment:
This is now the default in 2.2 rtm. Is this not reflected yet in the 2.2.0-rtm-35661 packages?